statsmodels/tsa/statespace/tests/test_tools.py

```diff
@@ -395,11 +396,12 @@ def test_cases(self):
                 cov = np.eye(unconstrained[0].shape[0])
             else:
                 cov = np.eye(unconstrained.shape[0])
-            constrained, _ = tools.constrain_stationary_multivariate(unconstrained, cov)
+            constrained, _ = tools.constrain_stationary_multivariate(unconstrained, cov)  # noqa:E501
             companion = tools.companion_matrix(
-                [1] + [-np.squeeze(constrained[i]) for i in range(len(constrained))]
+                [1] + [-np.squeeze(constrained[i])
+                       for i in range(len(constrained))]
             ).T
-            assert_equal(np.max(np.abs(np.linalg.eigvals(companion))) < 1, True)
+            assert np.max(np.abs(np.linalg.eigvals(companion))) < 1
```

Comment:
I have not been keeping up with the discussion on this, but I thought we didn't like plain assert statements?

Reply:
That's a relic from a long time ago, when running `python -A` (I think -A, not totally sure) would strip out assertions.  Now that we're using pytest, it rewrites assertions, so this more readable version is the way to go.

Reply:
@josef-pkt is this true? are `assert`s okay now?

Reply:
This one I'll consent to change to an assert_array_less.  I think that's the only one here.